packages/redux-query-react/flow-test/hooks/use-request-on-demand.js

```diff
@@ -15,7 +15,9 @@ const Card = () => {
       {isPending === true ? 'loading…' : 'loaded'}
       {/* $FlowFixMe expected */}
       <button onClick={() => request('false')}>Trigger</button>
```

Comment:
This is redundant with the example on line 20

Reply:
this is actually where the main test for an invalid type happens, the `{/* $FlowFixMe expected */}` expects a flow type error on the line below it.

lines 18, 19, 20 are a tad redundant, but IMO its helpful to have explicit examples here